packages/components/Checkbox/src/Checkbox.tsx

```diff
@@ -63,7 +54,7 @@ export const Checkbox = compose<CheckboxType>({
       return (
         <Slots.root {...mergedProps}>
           {Checkbox.state.labelIsBefore && labelComponent}
-          <Slots.checkbox>
+          <Slots.checkbox onPress={onPress} {...accessibilityState}>
```

Comment:
This changes the hit target for other platforms, which is not what we want. The hit target for checkbox should be the entire View wrapping the checkbox and the label on other platforms. AccessibilityState should also stay on the root view for other platforms.

Reply:
To confirm: 
This means that on win32, clicking on the label would also select it's checkbox?

Reply:
Yes, that's the expected behavior. This is also the behavior on web: https://react.fluentui.dev/?path=/docs/components-checkbox--default

Reply:
Changed completed and approved  , hence resolving.